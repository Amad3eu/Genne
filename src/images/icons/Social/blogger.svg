<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
<path style="fill:#FF7043;" d="M498.944,198.08c-10.816-4.608-57.184,0.512-70.048-11.104c-9.088-8.384-9.664-23.552-13.216-43.776
	c-5.952-33.888-8.416-41.568-14.592-54.912C378.592,40.736,324.704,0,275.744,0H162.24C72.96,0,0,72.864,0,161.824v188.672
	C0,439.296,72.96,512,162.24,512h186.464c89.28,0,161.76-72.704,162.272-161.504L512,219.84C512,219.84,512,203.68,498.944,198.08z
	 M164.288,132.256h89.984c17.152,0,31.072,13.856,31.072,30.784c0,16.96-13.92,30.944-31.072,30.944h-89.984
	c-17.152,0-31.072-14.016-31.072-30.944C133.216,146.08,147.136,132.256,164.288,132.256z M347.168,378.912h-182.88
	c-17.152,0-31.072-14.016-31.072-30.784c0-16.928,13.92-30.784,31.072-30.784h182.88c16.992,0,30.912,13.856,30.912,30.784
	C378.08,364.896,364.16,378.912,347.168,378.912z"/>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
