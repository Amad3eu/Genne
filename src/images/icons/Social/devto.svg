<svg viewBox="0 0 235 234" xmlns="http://www.w3.org/2000/svg" class="rainbow-logo" preserveAspectRatio="xMinYMin meet"><g fill="none" fill-rule="evenodd"><path fill="#88AEDC" d="m234.04 175.67-75.69 58.28h47.18L234.04 212z"/><path d="m234.04 140.06-121.93 93.89h.02l121.91-93.87zM133.25.95.04 103.51v.02L133.27.95z"/><path fill="#F58F8E" fill-rule="nonzero" d="M.04.95v30.16L39.21.95z"/><path fill="#FEE18A" fill-rule="nonzero" d="M39.21.95.04 31.11v35.9L85.84.95z"/><path fill="#F3F095" fill-rule="nonzero" d="M85.84.95.04 67.01v36.5L133.25.95z"/><path fill="#55C1AE" fill-rule="nonzero" d="M133.27.95.04 103.53v35.59L179.49.95z"/><path fill="#F7B3CE" fill-rule="nonzero" d="M234.04.95h-7.37L.04 175.45v35.93l234-180.18z"/><path fill="#88AEDC" fill-rule="nonzero" d="M179.49.95.04 139.12v36.33L226.67.95z"/><path fill="#F58F8E" fill-rule="nonzero" d="M234.04 31.2.04 211.38v22.57h18.03l215.97-166.3z"/><path fill="#FEE18A" fill-rule="nonzero" d="M234.04 67.65 18.07 233.95H64.7l169.34-130.39z"/><path fill="#F3F095" fill-rule="nonzero" d="M234.04 103.56 64.7 233.95h47.41l121.93-93.89z"/><path fill="#55C1AE" fill-rule="nonzero" d="m234.04 140.08-121.91 93.87h46.22l75.69-58.28z"/><path fill="#F7B3CE" fill-rule="nonzero" d="m234.04 212-28.51 21.95h28.51z"/><g fill="#FFF"><path d="M65.2372 77.75c4.5139.95 7.774 2.8 11.1343 6.3 3.0595 3.2 4.9653 6.85 5.7678 10.95.652 3.45.652 40.55 0 44.05-1.7052 9.1-9.4792 16.2-19.109 17.45-2.0062.25-8.7269.5-14.8458.5H37V77h12.4384c8.8272 0 13.3412.2 15.7988.75ZM51.5449 117v25.6l5.166-.2c4.4637-.15 5.4167-.35 7.4228-1.5 3.9121-2.3 3.9623-2.45 3.9623-24.2 0-21.2 0-21.2-3.6613-23.6-1.8056-1.2-2.558-1.35-7.473-1.55l-5.4168-.15V117ZM130.7895 84.25v7.25h-25.579v18h15.548V124h-15.548l.1003 9.1.1505 9.15 12.6892.15 12.639.1V157h-14.7957c-16.4508 0-19.0087-.3-21.6167-2.6-3.6613-3.2-3.4607-1.15-3.6112-36.3-.1003-21.9.0502-32.25.4013-33.65.7021-2.6 3.6613-5.8 6.2693-6.7 1.5548-.55 5.467-.7 17.7047-.75h15.6483v7.25ZM162.4372 105.1c3.7114 14.25 6.821 25.6 6.9213 25.25.1505-.35 3.3102-12.4 7.0718-26.85l6.8712-26.25 7.8242-.15c5.9183-.1 7.8743.05 7.8743.5s-17.3536 66.2-18.3567 69.5c-.7021 2.3-4.4637 7-6.5702 8.25-2.6582 1.5-6.5703 1.75-8.9777.5-2.1567-1.1-5.0155-4.4-6.47-7.5-.9028-1.9-15.6483-56-19.0588-70l-.3511-1.35h7.8241c7.6737 0 7.8743 0 8.2756 1.1.2006.65 3.4105 12.8 7.122 27Z"/></g></g></svg>