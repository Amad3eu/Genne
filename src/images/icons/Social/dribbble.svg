<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
<circle style="fill:#EC407A;" cx="256" cy="254.272" r="240"/>
<path style="fill:#C2185B;" d="M256,0C114.848,0,0,114.848,0,256s114.848,256,256,256s256-114.848,256-256S397.152,0,256,0z
	 M479.904,257.92c-54.208-14.304-109.792-15.968-162.752-5.056c-7.232-18.048-15.04-35.808-23.424-53.344
	c47.904-22.4,92.192-52.768,131.04-90.304C459.04,148.544,480,199.808,480,256C480,256.64,479.904,257.248,479.904,257.92z
	 M402.144,86.624c-36.288,35.04-77.568,63.584-122.304,84.448c-23.904-45.76-51.872-89.472-83.52-130.752
	C215.36,35.04,235.296,32,256,32C311.84,32,362.88,52.672,402.144,86.624z M164.704,51.648
	c32.48,41.344,60.992,85.376,85.28,131.488c-67.2,25.664-140.16,35.52-213.216,26.976C51.584,139.2,100.096,80.64,164.704,51.648z
	 M32,256c0-4.8,0.416-9.504,0.736-14.208c18.944,2.24,37.888,3.712,56.768,3.712c60.224,0,119.392-11.776,174.848-33.376
	c7.68,16,14.912,32.224,21.6,48.672c-78.656,23.616-148.736,73.984-195.904,145.12C54.112,366.176,32,313.664,32,256z
	 M113.44,428.64c43.392-68.224,109.568-116.32,184.032-138.08c20.224,55.68,34.784,113.504,43.264,172.672
	C314.592,473.984,285.984,480,256,480C201.888,480,152.192,460.704,113.44,428.64z M370.752,448
	c-8.8-56.384-22.752-111.584-41.856-164.864c48.352-9.088,98.944-6.944,148.224,6.976C466.784,357.184,426.752,414.4,370.752,448z"
	/>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
