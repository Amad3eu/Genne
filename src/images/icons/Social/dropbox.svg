<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
<g>
	<polygon style="fill:#1E88E5;" points="512,99.136 361.376,0 256,88.672 407.872,183.168 	"/>
	<polygon style="fill:#1E88E5;" points="150.624,0 0,99.104 104.16,183.168 256,88.672 	"/>
	<polygon style="fill:#1E88E5;" points="256,277.696 150.624,366.368 0,267.232 104.128,183.168 256,277.696 407.84,183.168 
		512,267.232 361.376,366.368 	"/>
	<path style="fill:#1E88E5;" d="M368.992,399.648c-2.656,1.76-5.728,2.624-8.768,2.624c-3.68,0-7.328-1.28-10.304-3.744L256,319.488
		l-93.92,79.04c-5.408,4.544-13.184,4.992-19.104,1.12L96,368.736V416l160.32,96L416,416v-47.296L368.992,399.648z"/>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
