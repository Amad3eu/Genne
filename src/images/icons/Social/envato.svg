<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 511.992 511.992" style="enable-background:new 0 0 511.992 511.992;" xml:space="preserve">
<path style="fill:#7CB342;" d="M411.809,2.806c-18.304-8.992-111.168,1.28-178.112,65.504
	C128.225,173.622,130.753,312.374,130.753,312.374s-3.488,14.272-18.336-6.432c-32.512-41.44-15.488-136.768-13.568-150.048
	c2.688-18.72-9.248-19.264-14.208-13.088C-32.863,305.942,73.249,439.51,142.721,481.91c81.344,49.632,241.792,49.6,305.696-91.2
	C528.033,215.382,434.753,14.038,411.809,2.806z"/>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
