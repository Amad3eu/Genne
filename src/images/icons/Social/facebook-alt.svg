<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
<path style="fill:#1976D2;" d="M384,176h-96v-64c0-17.664,14.336-32,32-32h32V0h-64l0,0c-53.024,0-96,42.976-96,96v80h-64v80h64v256
	h96V256h64L384,176z"/>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
