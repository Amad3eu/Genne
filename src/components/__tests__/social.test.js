import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Social from '../social';

describe('Social', () => {
  const mockEvent = { target: { value: 'This is a mock event' } };
  const props = {
    social: {
      github: 'github ',
      twitter: 'twitter',
      dev: 'dev',
      codepen: 'codepen',
      codesandbox: 'codesandbodx',
      stackoverflow: 'stackoverflow',
      linkedin: 'linkedin',
      kaggle: 'kaggle',
      fb: 'fb',
      instagram: 'instagram',
      dribble: 'dribble',
      behance: 'behance',
      medium: 'medium',
      youtube: 'youtube',
      codechef: 'codechef',
      hackerrack: 'hackerranck',
      codeforces: 'codeforces',
      leetcode: 'leetcode',
      topcoder: 'topcoder',
      hackerearth: '@hackerearth',
      geeks_for_geeks: 'geeks_for_geeks',
      discord: 'discord',
      rssurl: 'rssurl',
    },
    handleSocialChange: jest.fn().mockReturnValue({}),
  };
  it('renders correctly', () => {
    const component = shallow(<Social {...props} />);
    for (let i = 0; i < component.find('input').length; i++) {
      component.find('input').at(i).simulate('change', mockEvent);
    }
    expect(toJson(component)).toMatchSnapshot();
  });
});
