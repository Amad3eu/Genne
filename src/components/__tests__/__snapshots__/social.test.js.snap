// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social renders correctly 1`] = `
<div
  className="px-2 sm:px-6 mb-4"
>
  <div
    className="text-xl sm:text-2xl font-bold font-title mt-2 mb-2"
  >
    Social
  </div>
  <div
    className="flex flex-wrap justify-center items-center"
  >
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="github"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/github.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-1 sm:px-2 focus:border-blue-700"
        id="github"
        onChange={[Function]}
        placeholder="github username"
        value="github "
      />
    </div>
    <div
      className="w-1/2  flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="twitter"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/twitter.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="twitter"
        onChange={[Function]}
        placeholder="twitter username"
        value="twitter"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="dev.to"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/dev-dot-to.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="dev"
        onChange={[Function]}
        placeholder="dev.to username"
        value="dev"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="codepen"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/codepen.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="codepen"
        onChange={[Function]}
        placeholder="codepen username"
        value="codepen"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="codesandbox"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/codesandbox.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="codesandbox"
        onChange={[Function]}
        placeholder="codesandbox username"
        value="codesandbodx"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="stackoverflow"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/stackoverflow.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="stackoverflow"
        onChange={[Function]}
        placeholder="stackoverflow user ID"
        value="stackoverflow"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="linkedin"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/linkedin.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="linkedin"
        onChange={[Function]}
        placeholder="linkedin username"
        value="linkedin"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="kaggle"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/kaggle.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="kaggle"
        onChange={[Function]}
        placeholder="kaggle username"
        value="kaggle"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="facebook"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/facebook.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="fb"
        onChange={[Function]}
        placeholder="facebook username"
        value="fb"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="instagram"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/instagram.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="instagram"
        onChange={[Function]}
        placeholder="instagram username"
        value="instagram"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="dribbble"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/dribbble.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="dribbble"
        onChange={[Function]}
        placeholder="dribbble username"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="behance"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/behance.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="behance"
        onChange={[Function]}
        placeholder="behance username"
        value="behance"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="medium"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/medium.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="medium"
        onChange={[Function]}
        placeholder="medium username (with @)"
        value="medium"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="youtube"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/youtube.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="youtube"
        onChange={[Function]}
        placeholder="youtube channel name"
        value="youtube"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="codechef"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/codechef.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="codechef"
        onChange={[Function]}
        placeholder="codechef username"
        value="codechef"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="hackerrank"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/hackerrank.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="hackerrank"
        onChange={[Function]}
        placeholder="hackerrank username"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="codeforces"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/codeforces.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="codeforces"
        onChange={[Function]}
        placeholder="codeforces username"
        value="codeforces"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="leetcode"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/leetcode.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="leetcode"
        onChange={[Function]}
        placeholder="leetcode username"
        value="leetcode"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="topcoder"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/topcoder.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="topcoder"
        onChange={[Function]}
        placeholder="topcoder username"
        value="topcoder"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="hackerearth"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/hackerearth.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="hackerearth"
        onChange={[Function]}
        placeholder="hackerearth user (with @)"
        value="@hackerearth"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="geeksforgeeks"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/geeksforgeeks.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="geeksforgeeks"
        onChange={[Function]}
        placeholder="GFG (<username>/profile)"
        value="geeks_for_geeks"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="discord"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/discord.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="discord"
        onChange={[Function]}
        placeholder="discord invite (only code)"
        value="discord"
      />
    </div>
    <div
      className="w-1/2 flex justify-center items-center text-xxs sm:text-lg py-4 pr-2 sm:pr-0"
    >
      <img
        alt="rssfeed"
        className="w-6 h-6 sm:w-8 sm:h-8 mr-1 sm:mr-4"
        src="https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/rss.svg"
      />
      <input
        className="outline-none placeholder-gray-700 w-32 sm:w-1/2 border-t-0 border-l-0 border-r-0 border solid border-gray-900 py-1 px-2 focus:border-blue-700"
        id="rssurl"
        onChange={[Function]}
        placeholder="RSS feed URL"
        value="rssurl"
      />
    </div>
  </div>
</div>
`;
